core/util/src/main/java/org/arakhne/afc/progress/DefaultProgression.java

```diff
@@ -200,7 +200,7 @@ public int getMinimum() {
 	@Override
 	@SuppressWarnings("checkstyle:magicnumber")
 	public double getPercent() {
-		final double extent = this.max - this.min;
+		final double extent = this.max - (double) this.min;
```

Comment:
Yes. You are right.

On Tue, May 31, 2016 at 6:36 PM, Nicolas GAUD <notifications@github.com>
wrote:

> In
> core/util/src/main/java/org/arakhne/afc/progress/DefaultProgression.java
> <https://github.com/gallandarakhneorg/afc/pull/80#discussion_r65206870>:
>
> > @@ -200,7 +200,7 @@ public int getMinimum() {
> >  	@Override
> >  	@SuppressWarnings("checkstyle:magicnumber")
> >  	public double getPercent() {
> > -		final double extent = this.max - this.min;
> > +		final double extent = this.max - (double) this.min;
>
> Not good because you force the usage of the double addition instead using
> the addition between integer before casting the resulting integer into a
> double.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/gallandarakhneorg/afc/pull/80/files/fe903167563ac273585ed61bab3fb0c748e5eda0#r65206870>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AR09BHeXW0ug-poe8AVB4PHt-nAtr2z-ks5qHFVqgaJpZM4Iqp-H>
> .
>